### PR TITLE
User settings profile view

### PR DESCRIPTION
**Pull request by emanuelschuetze**
@ostcar: ready for debug

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
Fixed. Ready for review.

File: openslides/users/static/js/users/users.js [hunk 1]
```diff
@@ -122,6 +122,14 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
             }
         }
     })
+    .state('users.user.detail.profile', {
+        views: {
+            '@users.user.detail': {},
```
Comment (ostcar) [inline openslides/users/static/js/users/users.js]: Please delete this line. It does not work

File: openslides/users/static/js/users/users.js [hunk 2]
```diff
@@ -336,6 +344,20 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
     };
 })
 
+.controller('UserProfileCtrl', function($scope, $state, User, user) {
+    console.log("profile page");
```
Comment (ostcar) [inline openslides/users/static/js/users/users.js]: Please delete this line

File: openslides/users/static/js/users/users.js [hunk 3]
```diff
@@ -336,6 +344,20 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
...
+    $scope.user = user;  // autoupdate is not activated
+    console.log(user);
```
Comment (ostcar) [inline openslides/users/static/js/users/users.js]: Please remove this line

**Comment by emanuelschuetze** [comment]
Ready for last review.

**Pull request merged**